src/main/resources/bootstrap.yaml

```diff
@@ -0,0 +1,13 @@
+spring:
+  cloud:
+    propertiesvolume:
+#      properties are loaded from given paths
+      enabled: true
+#      parent directory prefixes the property names
+      prefixed: true
+      paths: /mnt/secrets/em-npa
+      aliases:
+        em-npa.FUNCTIONAL_TEST_CLIENT_S2S_TOKEN: FUNCTIONAL_TEST_CLIENT_S2S_TOKEN
```

Comment:
mapping should be from secrets to env vars. em-npa.{secretkey} : {envvar} is the syntax